metricbeat/module/rabbitmq/url.go

```diff
@@ -0,0 +1,26 @@
+package rabbitmq
+
+import (
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+// Subpaths to management plugin endpoints
+const (
+	ConnectionsPath = "/api/connections"
```

Comment:
I'm going forth an back if this should be in the module or in the metricset. Is there a benefit that the module knows about these paths?